doc/source/cluster/kubernetes/getting-started/rayjob-quick-start.md

```diff
@@ -63,6 +63,11 @@ To understand the following content better, you should understand the difference
   * `ttlSecondsAfterFinished` (Optional): Only works if `shutdownAfterJobFinishes` is true. The KubeRay operator deletes the RayCluster and the submitter `ttlSecondsAfterFinished` seconds after the Ray job finishes. The default value is 0.
   * `activeDeadlineSeconds` (Optional): If the RayJob doesn't transition the `JobDeploymentStatus` to `Complete` or `Failed` within `activeDeadlineSeconds`, the KubeRay operator transitions the `JobDeploymentStatus` to `Failed`, citing `DeadlineExceeded` as the reason.
   * `DELETE_RAYJOB_CR_AFTER_JOB_FINISHES` (Optional, added in version 1.2.0): Set this environment variable for the KubeRay operator, not the RayJob resource. If you set this environment variable to true, the RayJob custom resource itself is deleted if you also set `shutdownAfterJobFinishes` to true. Note that KubeRay deletes all resources created by the RayJob, including the Kubernetes Job.
+  * `deletionPolicy` (Optional, alpha in v1.3.0): Indicates what resources of the RayJob are deleted upon job completion. Valid values are `DeleteCluster`, `DeleteWorkers`, `DeleteSelf` or `DeleteNone`. If unset, deletion policy is based on `spec.shutdownAfterJobFinishes`. This field requires the `RayJobDeletionPolicy` feature gate to be enabled.
+    * `DeleteCluster` - Deletion policy to delete the RayCluster custom resource (and its Pods) on job completion.
```

Comment:
Ray uses Vale for documentation, and it checks some writing rules defined by Google. For example, it includes a rule that advises avoiding parentheses.

https://docs.ray.io/en/latest/ray-contribute/docs.html#how-to-use-vale

A lot of Ray docs don't follow the rules, but the doc team tries to make the new docs follow the rule if possible.

Reply:
ack -- I removed the parenthesis. 

> A lot of Ray docs don't follow the rules

This is my biggest gripe with usign vale, on most files it's cluttered with warnings that are difficult to resolve. It would be helpful to have a presubmit check that only runs `vale` against the diff.

Reply:
thanks